ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -567,9 +567,11 @@ registerInitialFunds initialFunds nes = nes {
           SL._reserves = reserves <-> coin (SL.balance initialFundsUtxo)
         }
 
```

Comment:
Style: some of these new `=` and adjacent old `=` ought to align, I think.

Reply:
Aligned!

Reply:
Line 576 and 577 too please :) `SL._utxo = ` and `SL.stakeDistro =`.

Reply:
(And the I script has opinions on your long `import` line.)